### PR TITLE
mesa: bump to 22.0.5

### PR DESCRIPTION
**Pull request by kenmays**
hrev56530 x64: Tested all included Haiku 3D screensavers/demos. No issues.

### CODE REVIEW

**Comment by BeagleJoe** [comment]
Tried this with hrev56387 and the game Speed Dreams.
Game flickers badly and has drawing artifacts.
Works fine with Mesa 21.3.7.

**Comment by kenmays** [comment]
Verified on hrev56443 x64. Same issue. Reviewing.

**Comment by kenmays** [comment]
Verified on hrev56443 x64. Same issue. Reviewing. Rebuild Mesa with '-Dshader-cache=true", then retest.

**Comment by kenmays** [comment]
Verified same issue with Mesa 22.1.7. Reverted to Mesa 21.3.8. 
- No issues with Blender3, Speed Dreams, Haiku demos, and Haiku screensavers tested.

**Comment by kenmays** [comment]
Bumped to Mesa 21.3.9.

No issues with Blender3, Speed Dreams, Haiku demos, and Haiku screensavers tested.

**Comment by kenmays** [comment]
Bumped to Mesa 22.0.5.
- No issues with Blender3, Speed Dreams, Haiku demos, and Haiku screensavers tested.
- Fixed: https://dev.haiku-os.org/ticket/14535

File: sys-libs/mesa/mesa-22.0.5.recipe [hunk 1]
```diff
@@ -122,6 +122,7 @@ BUILD()
 		-Dgallium-drivers=swrast \
 		-Dvulkan-drivers=swrast \
 		-Dvulkan-icd-dir=$addOnsDir/vulkan/icd.d \
+		-Dshader-cache=false \
```
Comment (waddlesplash) [inline sys-libs/mesa/mesa-22.0.5.recipe:125]: Why add this?

Comment (waddlesplash) [reply sys-libs/mesa/mesa-22.0.5.recipe:125]: The Intel driver does not interact with MESA in any way. Please remove this flag.

Comment (kenmays) [reply sys-libs/mesa/mesa-22.0.5.recipe]: Use MESA_SHADER_CACHE_DISABLE=1 if any app has an issue with it.

**Comment by waddlesplash** [comment]
Build failed: https://build.haiku-os.org/buildmaster/master/x86_64/?buildrunDir=7559&viewMode=expanded

**Pull request merged**